about.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" href="./images/favicon.ico" type="image/x-icon" />
+
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="about.css" />
+    <title>About</title>
+  </head>
+  <body>
+    <header>
+      <div id="first-header">
+        <img src="./images/logo-facebook.png" alt="" />
+        <img src="./images/logo-tweeter.png" alt="" />
+        <ul id="navbar-h1">
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </div>
+    </header>
+    <div id="second-header">
+      <a href="./index.html" id="logo-header"
+        ><img src="./images/logo.png" alt=""
+      /></a>
+      <ul id="navbar-h2">
+        <li><a href="./about.html">About</a></li>
+        <li><a href="#">Program</a></li>
+        <li><a href="#">Join</a></li>
+        <li><a href="#">Sponsor</a></li>
+        <li><a href="#">News</a></li>
+        <li><button>CC Campaign</button></li>
+      </ul>
+    </div>
+    <section id="section1">
+      <nav class="menu-responsive">
+        <div class="x-section">
+          <img src="./images/x-icon.png" alt="" />
+        </div>
+        <ul class="navbar-menu-responsive">
+          <li><a href="./index.html" class="items-menu-responsive">Home</a></li>
+          <li><a href="#" class="items-menu-responsive">Program</a></li>
+          <li><a href="#" class="items-menu-responsive">Join</a></li>
+          <li><a href="#" class="items-menu-responsive">Sponsor</a></li>
+          <li><a href="#" class="items-menu-responsive">News</a></li>
+          <li><button>CC Campaign</button></li>
+        </ul>
+      </nav>
+      <div id="main-banner">
+        <img id="hamburg" src="./images/menu.png" alt="" />
+        <div id="main-content">
+          <h4>Don't miss these trips!</h4>
+
+          <h1 id="text-image">CREATIVE COMMONS GLOBAL SUMMIT 2023</h1>
+          <hr class="line4" />
+        </div>
+        <div id="first-parraf-banner">
+          <p class="origin-p">
+            We have been working together since 2015 providing our clients with
+            an unforgettable experience accompanying them on this incredible
+            journey into the world of live music, thousands of people have
+            already tried our service around the world.

```

Comment:
- [ ] Instead of using a  global `<section>` tag to wrap all the main content of your page, I suggest using the `<main>` tag for this purpose, and using `<section>` tags to separate each of the standalone sections of your page. Something like this:

```html
<main>
  <section id="main-banner">
    <!-- HTML for the main banner here -->
  </section>

  <section id="creative-commons">
    <!-- HTML for logo here -->
  </section>
  ...
</main>
```

This way you can improve the semantic structure of your page, and improve its readability! 🤓 🔧  


⚠️ **Consider applying these changes to your `index.html` too!** ⚠️